content/applications/finance/accounting/bank/reconciliation/bank_reconciliation.rst

```diff
@@ -0,0 +1,90 @@
+===================
+Bank reconciliation
+===================
+
+Introduction
+============
+
+Bank reconciliation is the process of linking *-or matching-* a registered payment with its
+corresponding bank statement. In Odoo, when a registered payment is reconciled with a bill or
+invoice, the status of the document changes to **In Payment** rather than **Paid**. It changes to
+**Paid** only once the registered payment has been reconciled with a bank statement. Odoo uses a
+transitory account called **suspense account** for unreconciled payments, and automatically changes
+the account once matched with a bank statement.
+
+.. note::
+   - Versions below V14 do not feature a **suspense account** and therefore directly mark a bill or
+     an invoice as **Paid** when registering a payment.
+
+.. image:: use_cases/payment-status.png
+   :align: center
+   :alt: The two different payment status ribbon.
+
+Entries can be reconciled either manually or automatically, depending on your needs. If you wish to
+automate your reconciliation process, you can refer to `our reconciliation models documentation <htt
+ps://www.odoo.com/documentation/15.0/applications/finance/accounting/bank/reconciliation/reconciliat
+ion_models.html>`_.
+
+.. seealso::
+   - :doc:`../feeds/bank_synchronization`
+   - `Odoo Tutorials: Customer and Vendor Payments
+     <https://www.odoo.com/slides/slide/customer-and-vendor-payments-1690?fullscreen=1>`_.
+
+.. note::
+   - If you wish, you can :ref:`bypass the "In Payment" status <bypass-in-payment>`.
+
+Payment registration
+====================
+
+If you have received a payment confirmation from your customer, or if you have made a payment to one
+of your vendors, click on :guilabel:`Register Payment` from within the related invoice or bill. Once
+you click, a pop-up window will appear prompting you to input the payment method and other
```

Comment:
```suggestion
you click, a pop-up window appears prompting you to input the payment method and other
```
As per the [content guidelines](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#grammatical-tenses), we should use the present tense.